requirements-dev.txt

```diff
@@ -5,11 +5,12 @@ tensorflow
 keras
 coremltools
 pandas
-pytest
 pytest-cov
 scikit-learn
 scipy
 svm
 wheel
 xgboost
 lightgbm
+pyspark==2.3.2
```

Comment:
any reason we have to stick to this version?

Reply:
version 2.4 does not work. Until they provide fixes for that yes we need to stick with this